onyxia-api/src/main/java/fr/insee/onyxia/api/services/impl/HelmAppsService.java

```diff
@@ -410,6 +409,96 @@ private Service getHelmApp(Region region, User user, HelmLs release) {
         return service;
     }
 
+    @Override
+    public void pause(
+            Region region,
+            Project project,
+            String catalogId,
+            Pkg pkg,
+            User user,
+            String serviceId,
+            boolean skipTlsVerify,
+            String caFile,
+            boolean dryRun)
```

Comment:
Yeah, that would make a lot of sense. Got a headache yesterday working on parameters overloading :laughing:

Reply:
Another suggestion could be to have one method taking in a status, instead of having two methods (`pause` and `resume`).
If that make sense?
E.g.: `public void update(Region region, ... , Status status` where status is an enum (PAUSE,RESUME), i.e.. Maybe it could be possible to further refactor such that delete also is a part of that enum.
What do you think?